test/test_pty.rs

```diff
@@ -0,0 +1,36 @@
+use nix::pty::openpty;
+use nix::unistd::{read, write, close};
+
+#[test]
+fn test_openpty() {
+    // TODO: figure out the right termios settings to pass in here
```

Comment:
Is it okay that `Default` won't necessarily set the fields to valid values? I have no idea if zero is a valid/correct default value for all the fields